pkg/cache/session_store.go

```diff
@@ -0,0 +1,46 @@
+package cache
+
+import (
+	"time"
+
+	"github.com/CESARBR/knot-babeltower/pkg/network"
+)
+
+// Cacher represents the operations to send commands response
```

Comment:
This docstring doesn't seem to be from SessionStore... could you update it? :smile: